.envrc

```diff
@@ -20,7 +19,7 @@ export SIMPLE_TEST_MODE="yes" # Reduces the amount of generated `relation_link_t
 
 ### QE specific logging vars ###
 export QE_LOG_LEVEL=debug # Set it to "trace" to enable query-graph debugging logs
-export FMT_SQL=1 # Uncomment it to enable logging formatted SQL queries
+# export FMT_SQL=1 # Uncomment it to enable logging formatted SQL queries
```

Comment:
Align the reality with the comment.

Reply:
```
# export PRISMA_RENDER_DOT_FILE=1 # Uncomment to enable rendering a dot file of the Query Graph from an executed query.
```

I don't know if that's a mistake but the env var above was completely removed

Reply:
ouch, looks like a bad rebase, I'll fix that tomorrow